demo/wp-feature-api-demo/includes/BootstrapAssets.php

```diff
@@ -21,12 +21,15 @@ public function enqueue_scripts() {
             true
         );
 
-		wp_enqueue_style(
-			'wp-components-css',
-			includes_url('css/dist/components/style.min.css'),
-			array(),
-			false
-		);
+		// Only enqueue wp-components CSS if it's not already loaded by core (e.g., on editor screens), otherwise things will not render correctly.
+		if ( ! wp_style_is( 'wp-components', 'enqueued' ) ) {
```

Comment:
It would be good to have dependencies automatically managed via  $asset_file['dependencies'], as we do for JS. But for now this seems to be the way to go.